### ISSUE TITLE
failure to import key should have a higher logging level than INFO

### ISSUE THREAD

**Issue by cyc115** [docker/notary#1178]
At the moment importing malformed keys with the notary client raises no error. using the debug flag `-D` we can see the error is being logged at INFO level.

eg. 
```
notary key import priv.key -D
> DEBU[0000] Using the following trust directory: /Users/XXX/.docker/trust
> INFO[0000] failed to import key to store: Invalid key generated, key may be encrypted and does not contain path header
```

**Comment by endophage** [comment]
looks like this was closed by #1179?

**Comment by riyazdf** [comment]
@endophage: 👍  yes!
